hack/buildconfig-template.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Removing in favor of https://github.com/openshift/cluster-logging-operator/pull/89/files#diff-4c7729766fb0e3362bb4ecdd4880efbeR1